.github/workflows/ci.yml

```diff
@@ -14,33 +14,33 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php-version: [ '7.4', '8.2' ]
+                php-version: [ '8.1', '8.4' ]
                 db-type: [ sqlite, mysql, pgsql, agnostic ]
                 symfony-version: [ '5-min', '5-max', '6-min', '6-max', '7-min', '7-max']
                 exclude:
```

Comment:
What's all this exclude mess - can it be removed?

Reply:
Good question. It's probably just to make sure that Symfony version is not run with an incompatible version. With 8.1, there is no problem, then again, I am not sure if there is a benefit in running every Symfony version with all PHP versions. After all, we are testing Propel, not Symfony. As someone who frequently waits for those tests to finish, I'd prefer to not run pointless tests.
Ultimately, we want to know if our code runs withPHP  8.1 and 8.4, and we want to know if it runs with Symfony min/max version of 5/6/7 (I think 5-min could be dropped). Not sure if we learn something interesting (or anything really) if we fully permutate this through.
Am I missing something there?

Reply:
I don't think you're missing anything here.  Perpl shouldn't be so tightly coupled with Symfony.  It's nice to have some tests, but it's shouldn't be such a primary part of the testing or development IMO.  

We do use Symfony framework for some of our stack (sf 1.4 actually), but Propel doesn't touch it and it's entirely irrelevant.  I'm sure that's not the case for many, as they're relying on Symfony for their console commands (we have our own custom ones).